packages/osd-babel-preset/node_preset.js

```diff
@@ -66,8 +66,8 @@ module.exports = (_, options = {}) => {
       [
         require.resolve('babel-plugin-module-resolver'),
         {
-          root: [path.resolve(__dirname, '../..')],
-          cwd: path.resolve(__dirname, '../..'),
+          root: options['babel-plugin-module-resolver']?.root ?? [path.resolve(__dirname, '../..')],
```

Comment:
Do we need to use `[]` wrap `options['babel-plugin-module-resolver']?.root` if exists?

Reply:
May not necessary, root can be an array or string, the plugin will normalize the option